QuizApp/Quiz/QuizListViews/QuizListView.swift

```diff
@@ -12,49 +12,52 @@ struct QuizListView: View {
     }
 
     var body: some View {
-        ScrollView {
-            VStack(alignment: .leading, spacing: 20) {
-                Picker("", selection: $segmentationSelection) {
-                    let items = ["All"] + CategorySection.allCases.map {$0.rawValue}
-                    ForEach(items, id: \.self) { option in
-                        Text(option)
-                    }
-                }
-                .onChange(of: segmentationSelection) { _ in
-                    getQuizzes()
+        VStack {
```

Comment:
Will this be updated with my previous comment or is there some new logic here?

Reply:
It's updated with the previous comment.